README.md

```diff
@@ -1,2 +1,31 @@
 # to-start-case
 Convert a string to Start Case for human display purposes.
+
+Converts a string, including strings in camelCase or snake_case, into Start Case (a variant
+of Title Case where all words start with a capital letter), it keeps original single quote
```

Comment:
The last part of this documentation (after the comma) should be a separate sentence and very strongly note that "UNLIKE OTHER Start Case libs: ... "